app/models/concerns/permissions_concern.rb

```diff
@@ -0,0 +1,12 @@
+module PermissionsConcern
+  extend ActiveSupport::Concern
+
+  def is_admin?
+    self.user_type == "administrator"
```

Comment:
Does the admin have any permissions yet?

Reply:
yes with user administration